docs/dev.rst

```diff
@@ -328,58 +287,25 @@ On macOS you can set, in your ``.env``:
 
 .. code-block:: shell
 
-    DJANGO_ES_URLS=http://docker.for.mac.host.internal:9200
+   DJANGO_ES_URLS=http://docker.for.mac.host.internal:9200
 
 Now, the Python inside Docker will connect to the Elasticsearch running on your host.
 
+pytest with coverage
+====================
 
-Bors
-====
-
-All PRs should be merged via the `bors merge bot <https://bors.tech>`_. Bors
-will automate that the requirements for a PR have been met, and will then
-merge the PR in an orderly fashion.
-
-Only users with write access to the repository may use bors. Other users will
-get an error message. To use bors to merge a PR, leave a comment that
-includes a line such as::
-
-    bors r+
-
-Alternatively, you can list the user that approved the PR, which could be
-someone else, or multiple people, such as one of the following::
-
-    bors r= @alex
-    bors r= @bob, @carol
-
-If a PR should not be merged, for example because it is a work-in-progress,
-then add the label ``bors-dont-merge`` to the PR. This will prevent bors from
-merging the PR, even if it is ``r+``ed. To allow bors to merge the PR again,
-remove the label and say ``bors r+`` again.
-
-It is possible to temporarily delegate permission to a user to approve a
-particular PR. For example, if the PR is documentation for the ops team, you
-could grant merge access to the ops engineer reviewing the documentation.
-Note that delegating to a use that already has permission (such as an admin
-of the repo) has no affect. To do so, use a command such as::
-
-    bors delegate= @ops-dave
-
-If a PR failed to merge for an intermittent reason, such as network failure,
-you can instruct bors to try to merge the same commit with the same approver
-again with the command::
-
-    bors retry
-
-For more details, see `the bors reference docs <https://bors.tech/documentation/>`_
```

Comment:
I'm pretty sure bors isn't working anymore. On top of that, it's not a service I use, so I figured I'd just nix it rather than try to figure out what's wrong. Future maintainers can change their minds on this if they like.

Reply:
Not working? When did it break?

Also, when I was on the Product Delivery team, they pushed for bors and managed to make it a documented best practice. I was reluctant at first but quite started to like it. It's not without disadvantages but it sure is nice to know that all merges are linear so that CI isn't green for 2 PRs on Monday which both break each other on Tuesday when merged. 

Also, with bors, you don't need to enable that thing in GitHub that forces that all PRs need to be up-to-date which became cumbersome sometimes. 

Having said that, sometimes you get cryptic problems from bors which I never figured out so I just typed "bors retry" till it worked. Kinda.

Reply:
I have no idea when it broke.

bors nicely solves the problem of requiring ff-merge-only and probably works way better than doing it manually as the number of contributors gets bigger. I prefer to do it manually and not have to deal with yet-another-service. That may change, but for now, there it is.

Reply:
For the record, one thing which is nice is that you can make a decision about a PR *whilst waiting for CI to complete* and then you don't have to come back. 

Without bors I often look at the PR first for a review, then I come back X minutes later to do the merge stuff. I know I'm supposed to context switch onto something else but sometimes it's hard to let go and you end up doing junk/twitter/social media during those X minutes instead of focusing on something better. 

Having said all that, I do like to run my own little `gg cleanup` script locally which takes care of deleting the branch and remove it from my fork too. And I can only run that once I know it's really merged into master.